.travis.yml

```diff
@@ -7,9 +7,9 @@ python:
   - "3.5"
 
 env:
-  - DJANGO="Django>=1.11,<2.0"
-  - DJANGO="Django>=2.0,<2.1"
-  - DJANGO="Django>=2.1,<2.2"
+  - DJANGO="Django==1.11.*"
+  - DJANGO="Django==2.1.*"
+  - DJANGO="Django==2.2.*"
```

Comment:
I've not seen the * syntax used here before.

Reply:
c.f. [PEP 440: Version Matching](https://www.python.org/dev/peps/pep-0440/#version-matching)

`2.2.*` would match e.g. `2.2rc1` but I can't see a problem there.

Reply:
> I've not seen the * syntax used here before.

I'm always getting confused by `>x,<y`, so I decided to change that and expected it to get rejected if unwanted.  `~=2.2.0` would have been better regarding rc releases, but is also harder to read for people new to programming or semver.

> would match `2.2rc1`

Oh, I was not aware of that. :/

Reply:
Not a problem. We're all learning. 🙂

Thanks for the help!